lib/ansible/modules/packaging/os/pacman.py

```diff
@@ -176,6 +176,15 @@ def get_version(pacman_output):
     return None
 
 
+def get_name(pacman_output):
+    """Take pacman -Qi or pacman -Si output and get the package name"""
+    lines = pacman_output.split('\n')
+    for line in lines:
+        if 'Name' in line:
```

Comment:
```suggestion
        if line.startswith('Name '):
```

Reply:
Yes, much better. Should be in the version getting function too. I'll add it there.